### PR TITLE
[bettercap] update to version 2.32.0

### PR DESCRIPTION
**Pull request by etmatrix**
Now bettercap require library libnetfilter_queue already submitted

### CODE REVIEW

File: packages/bettercap/build.sh [hunk 1]
```diff
@@ -2,11 +2,11 @@ TERMUX_PKG_HOMEPAGE=https://www.bettercap.org
...
-TERMUX_PKG_VERSION=2.29
+TERMUX_PKG_VERSION=2.32.0
 TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=https://github.com/bettercap/bettercap/archive/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=c414be98a48d9f279460b325dddaef8479132a1470c6f305790d79e04dac1297
-TERMUX_PKG_DEPENDS="libpcap, libusb"
+TERMUX_PKG_SHA256=ea28d4d533776a328a54723a74101d1720016ffe7d434bf1d7ab222adb397ac6
+TERMUX_PKG_DEPENDS="libpcap, libusb, libnetfilter-queue"
 # x86_64 seem to depend on libnetfilter_queue
 TERMUX_PKG_BLACKLISTED_ARCHES="x86_64"
```
Comment (Grimler91) [inline packages/bettercap/build.sh:11]: We can hopefully remove blacklisting of x86_64 now as well

**Comment by OldiLo** [comment]
Good

**Comment by Grimler91** [comment]
Nice work, thanks!

**Comment by etmatrix** [comment]
I forget the IPTOS_TOS I'm fixing

**Comment by etmatrix** [comment]
After the pull request #273 bettercap should run

**Pull request merged**